tools/odbc/test/tests/time.cpp

```diff
@@ -0,0 +1,228 @@
+#include "../common.h"
+
+using namespace odbc_test;
+
+void test_row_column_count(SQLHANDLE stmt) {
+
+	SQLRETURN ret;
+
+	SQLLEN row_count;
+	EXECUTE_AND_CHECK("SQLRowCount()", SQLRowCount, stmt, &row_count);
+	REQUIRE(row_count == -1);
+
+	SQLSMALLINT col_count;
+	EXECUTE_AND_CHECK("SQLNumResultCols()", SQLNumResultCols, stmt, &col_count);
+	REQUIRE(col_count == 1);
+}
+
+//**************************************************************************
+TEST_CASE("Get SQL TIMESTAMP value via SQLGetData", "[odbc][time]") {
+	SQLRETURN ret;
+	SQLHANDLE env;
+	SQLHANDLE dbc;
+	SQLHANDLE stmt;
+
+	CONNECT_TO_DATABASE(env, dbc);
+
+	EXECUTE_AND_CHECK("SQLAllocHandle()", SQLAllocHandle, SQL_HANDLE_STMT, dbc, &stmt);
+
+	EXECUTE_AND_CHECK("SQLExecDirect()", SQLExecDirect, stmt,
+	                  (SQLCHAR *)"SELECT TIMESTAMP '1992-09-20 12:34:56.123456' as dt", SQL_NTS);
+
+	test_row_column_count(stmt);
+
+	SQL_TIMESTAMP_STRUCT ts_val;
+	SQLSMALLINT buffer_length = 0;
+	unsigned char target_value[4096];
+	SQLSMALLINT string_length = 0;
+	SQLLEN numeric_attribute;
+
+	EXECUTE_AND_CHECK("SQLColAttribute()", SQLColAttribute, stmt, 1, 2, &target_value, buffer_length, &string_length,
+	                  &numeric_attribute);
+	REQUIRE(numeric_attribute == 93);
```

Comment:
Instead of using `93` can you replace it with the keyword instead (`SQL_TIMESTAMP`).  The same goes for the `2` in SQLColAttribute.  This just makes it more clear what we're actually testing.